contrib/aws/rigger/create.sh

```diff
@@ -0,0 +1,116 @@
+function aws-setup-keypair {
+  local deis_auth_key="${1}"
+
+  rerun_log "Importing ${deis_auth_key} keypair to EC2"
+
+  # TODO: don't hardcode --key-names
+  if ! aws ec2 describe-key-pairs --key-names "deis" >/dev/null ; then
+    rerun_log "Importing ${deis_auth_key} keypair to EC2"
+    aws ec2 import-key-pair --key-name deis \
+        --public-key-material file://${deis_auth_key}.pub \
+        --output text
+  fi
+}
+
+function aws-provision-cluster {
+  local stack_name="${1}"
+
+  # customize cloudformation.json to use m3.medium instances
+  cat > $DEIS_ROOT/contrib/aws/cloudformation.json <<EOF
+  [
+      {
+          "ParameterKey":     "KeyPair",
+          "ParameterValue":   "deis"
+      },
+      {
+          "ParameterKey":     "InstanceType",
+          "ParameterValue":   "m3.medium"
```

Comment:
Unless this is test-specific provisioning code, we should leave the default instance size of `m3.large`. We try to use smaller instances to save a bit of money when testing, but we recommend users provision with `m3.large` or better.